bundle/tests/scorecard/minikube-kuttl/ingress/00-assert.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ingress
+status:
+  readyReplicas: 1
+---
+apiVersion: extensions/v1beta1
```

Comment:
operator generates ingress with apiVersion `networking.k8s.io/v1`
Is this minikube specific?

Reply:
Hmm.. `apiVersion: extensions/v1beta1` is what gets created in the Minikube cluster when expose is set to True. Minikube lists both `networking.k8s.io/v1` and `extensions/v1beta1` as supported.